.github/workflows/v3.yml

```
@@ -66,9 +63,11 @@ jobs:
           wait-on-timeout: 300
           # only record the results to dashboard.cypress.io if CYPRESS_RECORD_KEY is set
           record: ${{ !!secrets.CYPRESS_RECORD_KEY }}
-          # only do parallel if we have a record key
-          parallel: ${{ !!secrets.CYPRESS_RECORD_KEY }}
           browser: chrome
+          # only run a single test
+          # TODO: setup a special test for this purpose
+          spec: cypress/e2e/import-v2.spec.ts
```

```suggestion
          spec: cypress/e2e/tool-shelf.spec.ts
```
Not that it matters much at this stage, but the tool-shelf spec might be a better choice for a starting smoke test. At least it creates one of every kind of component.